xidmap/xidmap.go

```diff
@@ -136,7 +136,10 @@ func New(opts XidMapOptions) *XidMap {
 		})
 		x.Check(err)
 	}
-	xm.zc = pb.NewZeroClient(opts.UidAssigner)
+
```

Comment:
Do we need this? I am guessing it might be needed for running bulk loader?

Reply:
Can we rewrite the code such that it's a function that gives uids, and it uses zero if it's present or alpha if it's present? Instead of adding the logic everywhere it's used.